app/screens/post_options/post_options.tsx

```diff
@@ -63,17 +72,61 @@ const PostOptions = ({
     const shouldRenderFollow = !(sourceScreen !== Screens.CHANNEL || !thread);
     const shouldShowBindings = bindings.length > 0 && !isSystemPost;
 
-    const snapPoints = [
+    const snapPoints = useMemo(() => {
+        const items: BottomSheetProps['snapPoints'] = [1];
+        let count = shouldShowBindings ? 0.5 : 0;
+        if (canReply && sourceScreen !== Screens.THREAD) {
+            count++;
+        }
+
+        if (shouldRenderFollow) {
+            count++;
+        }
+
+        if (canMarkAsUnread && !isSystemPost) {
+            count++;
+        }
+
+        if (canCopyPermalink) {
+            count++;
+        }
+
+        if (!isSystemPost) {
+            count++;
+        }
+
+        if (canCopyText && post.message) {
```

Comment:
```suggestion
        if (canCopyText) {
```
`post.messaage` is already check for in the `canCopyText` variable assignment.

Reply:
<img width="883" alt="image" src="https://user-images.githubusercontent.com/7575921/210711955-f7c0d595-8b9f-4171-ada1-1dad6fc01ffe.png">